src/gluonts/ev/metrics.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass
+from functools import partial
+from typing import Optional
+
+import numpy as np
+
+from gluonts.exceptions import GluonTSUserError
+from .api import (
+    DerivedMetricEvaluator,
+    MetricEvaluator,
+    StandardMetricEvaluator,
+)
+from .aggregations import Mean, Sum
+from .stats import (
+    absolute_error,
+    absolute_label,
+    absolute_percentage_error,
+    absolute_scaled_error,
+    coverage,
+    quantile_loss,
+    scaled_interval_score,
+    squared_error,
+    symmetric_absolute_percentage_error,
+)
+
+
+def assert_axis_is_None_or_one(axis: Optional[int], metric_name: str) -> None:
+    if axis not in [None, 1]:
+        raise GluonTSUserError(
+            f"'{metric_name}' requires 'axis' to be None or 1 (not {axis})"
+        )
+
+
+@dataclass
+class AbsoluteLabelMean:
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=absolute_label,
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class AbsoluteLabelSum:
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=absolute_label,
+            aggregate=Sum(axis=axis),
+        )
+
+
+@dataclass
+class AbsoluteErrorSum:
+    forecast_type: str = "0.5"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=partial(absolute_error, forecast_type=self.forecast_type),
+            aggregate=Sum(axis=axis),
+        )
+
+
+@dataclass
+class MeanSquaredError:
+    forecast_type: str = "mean"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=partial(squared_error, forecast_type=self.forecast_type),
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class QuantileLoss:
+    q: float = 0.5
```

Comment:
I don't think we want a default value here.

Reply:
Should this be `SumQuantileLoss`?

Reply:
> I don't think we want a default value here.
Alright. 👍 

Yes, we can put the "Sum" in the name. Maybe `QuantileLossSum` sounds more natural than `SumQuantileLoss`?

Reply:
Whatever is most consistent.

Reply:
I actually went with your idea now. 😄 If we put "Mean" in the beginning (MeanSquaredError etc.), it makes sense to start with "Sum" as well.